### PR TITLE
Remove obsolete features from ShellInterface

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
Do not merge this yet.

**Comment by axic** [comment]
@chfast ready for review?

File: src/eei.cpp [hunk 1]
```diff
@@ -56,6 +56,12 @@ string toHex(evm_uint256be const& value) {
 
 }
 
+  void EthereumInterface::importGlobals(std::map<Name, Literal>& globals, Module& wasm) {
+    (void)globals;
+    (void)wasm;
+    HERA_DEBUG << "importGlobal " << "\n";
```
Comment (chfast) [inline src/eei.cpp]: Why 2 literals?

Comment (axic) [reply src/eei.cpp]: Because there was `<< globals <<` in between :)

If you know an easy way to dump a map that would be nice.

Comment (chfast) [reply src/eei.cpp]: Only `for` loop.

Comment (axic) [reply src/eei.cpp]: So should this be updated?

**Comment by axic** [comment]
@chfast can this be merged now?

**Pull request merged**